src/overview/tabs/Overview/cards/VmMigrationsDonutCard.tsx

```diff
@@ -73,63 +72,69 @@ const VmMigrationsDonutCard: FC<VmMigrationsDonutCardProps> = () => {
       </CardHeader>
       <CardBody className="forklift-overview__status-migration pf-v5-u-display-flex pf-v5-u-align-items-center pf-v5-u-flex-direction-column">
         <div className="forklift-overview__status-migration-donut">
-          <ChartDonut
-            ariaDesc={t('Donut chart with VM migration statistics')}
-            colorScale={colorScale}
-            constrainToVisibleArea
-            data={
-              vmCount.Total === 0
-                ? [{ x: 'Empty state', y: 1 }]
-                : ([
-                    { x: t('running'), y: vmCount.Running },
-                    { x: t('failed'), y: vmCount.Failed },
-                    { x: t('succeeded'), y: vmCount.Succeeded },
-                    { x: t('canceled'), y: vmCount.Canceled },
-                  ] as ChartDatum[])
-            }
-            labels={({ datum }: { datum: ChartDatum }) =>
-              vmCount.Total === 0
-                ? (undefined as unknown as string)
-                : `${t('{{count}} VM migration', { count: datum.y })}
+          <LoadingSuspend obj={obj} loaded={loaded} loadError={loadError}>
+            <ChartDonut
+              ariaDesc={t('Donut chart with VM migration statistics')}
+              colorScale={colorScale}
+              constrainToVisibleArea
+              data={
+                total === 0
+                  ? [{ x: 'Empty state', y: 1 }]
+                  : ([
+                      { x: t('running'), y: totalRunningCount },
+                      { x: t('failed'), y: totalFailedCount },
+                      { x: t('succeeded'), y: totalSucceededCount },
+                      { x: t('canceled'), y: totalCanceledCount },
```

Comment:
the x label should start with capital letter?

Reply:
for the donut chart, the x value will only be visible in the label, which looks like this: y VM migrations x